src/map/time_server.cpp

```diff
@@ -87,6 +87,28 @@ int32 time_server(time_point tick,CTaskMgr::CTask* PTask)
         }
     }
 
+    //JST Midnight
+    static time_point lastTickedJstMidnight;
+    if (CVanaTime::getInstance()->getJstHour() == 0 && CVanaTime::getInstance()->getJstMinute() == 0)
+    {
+        if (tick > (lastTickedJstMidnight + 1h))
+        {
+            // roeutils::CycleDailyRecords();
```

Comment:
These commented lines are left to avoid future merge conflicts if other functions are moved into these areas before RoE Phase 3 is ready.